app/services/billable_metrics/prorated_aggregations/base_service.rb

```diff
@@ -27,33 +27,33 @@ def compute_pay_in_advance_aggregation
 
         value = (result_without_proration * proration_coefficient).ceil(5)
 
-        extend_event_metadata(value)
+        extend_cached_aggregation(value)
 
         value
       end
 
-      # We need to extend event metadata with max_aggregation_with_proration. This attribute will be used
+      # We need to extend cached aggregation with max_aggregation_with_proration. This attribute will be used
       # for current usage in pay_in_advance case
-      def extend_event_metadata(prorated_value)
+      def extend_cached_aggregation(prorated_value)
         result.max_aggregation = aggregation_without_proration.max_aggregation
         result.current_aggregation = aggregation_without_proration.current_aggregation
 
-        unless previous_event
+        unless cached_aggregation
           result.max_aggregation_with_proration = prorated_value.to_s
 
           return
         end
 
         if BigDecimal(aggregation_without_proration.max_aggregation) >
-           BigDecimal(previous_event.metadata['max_aggregation'])
+           BigDecimal(cached_aggregation.max_aggregation)
```

Comment:
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.